django_react_proj/users-fe/src/components/TaskLegend/TypesLegend.module.css

```diff
@@ -0,0 +1,37 @@
+.TypesLegend{
+    box-shadow: rgba(50, 50, 105, 0.15) 0px 2px 5px 0px, rgba(0, 0, 0, 0.05) 0px 1px 1px 0px;
+    width: 20%;
+}
+
+.Meeting {
+    width:10px;
```

Comment:
Those classes can be unified into single class, e.g. `taskTypeCircle` which provides common css attributes and then `Email` `Housework` `Meeting` would provide only color. You can use as many classes as you want

`class="class1 class2"` which for your case would be something like 
```
className={`${styles.taskTypeCircle} ${styles.Email}`}
```
 which is not that good loking, but you can provide utility function or just use already written one which is: https://www.npmjs.com/package/classnames